diff -Naur pwr-button-handler_2014.42/debian/rules pwr-button-handler_2014.42a/debian/rules
--- a/debian/rules	2015-01-13 18:51:03.912772222 +0000
+++ b/debian/rules	2015-01-13 18:58:04.937345660 +0000
@@ -19,7 +19,7 @@
 
 # main packaging script based on dh7 syntax
 %:
-	dh $@ 
+	dh $@ --with=systemd
 
 # debmake generated override targets
 # This is example for Cmake (See http://bugs.debian.org/641051 )
